RadixWallet/SargonExtensions/SargonHostInteractor.swift

```diff
@@ -114,4 +114,22 @@ final class SargonHostInteractor: HostInteractor {
 			fatalError("Unexpected action")
 		}
 	}
+
+	func spotCheck(factorSource: FactorSource, allowSkip: Bool) async throws -> SpotCheckResponse {
+		let action = await overlayWindowClient.spotCheck(factorSource: factorSource, allowSkip: allowSkip)
```

Comment:
nit: I think `performedAction`, `executed` or `outcome` is slightly better, to make it clear that the action **has happened**. (which is clear from `.cancelled`).